src/Sulu/Bundle/PageBundle/Search/EventSubscriber/BlameTimestampSubscriber.php

```diff
@@ -95,34 +95,27 @@ private function getFieldValue($document, $fieldName)
     /**
      * Map timestamps to the search document.
      */
-    private function mapTimestamp(Document $document, ?\DateTime $created = null, ?\DateTime $changed = null)
-    {
-        $document->addField(
-            $this->factory->createField('created', $created ? $created->format('c') : null, 'string')
-        );
-
-        $document->addField(
-            $this->factory->createField('changed', $changed ? $changed->format('c') : null, 'string')
-        );
+    private function mapTimestamp(
```

Comment:
here are no changes? then I would revert it

Reply:
The signiture changed, from `DateTime` to `DateTimeInterface`

Reply:
okay reverted the other part as it did technical not change to make this things easier to understand what we changed here to support it